Makefile.devcontainer

```diff
@@ -75,7 +75,8 @@ FILES = $(shell find . -type f -name '*.go')
 
 format:                         ## Format source code.
 	gofumpt -l -w $(FILES)
-	goimports -local github.com/percona/pmm-managed -l -w $(FILES)
+        goimports -local github.com/percona/pmm-managed -l -w $(FILES)
+	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
```

Comment:
Redundand tab space before goimports. But do we need goimports at all now?

Reply:
old comment so not if still valid, but looks like we do (https://github.com/daixiang0/gci/issues/1#issuecomment-668305417)

Reply:
Aha, got it! Thanks!